### PR TITLE
fix:set player currentTime

### PR DESCRIPTION
**Pull request by RamK777-stack**

### CODE REVIEW

**Comment by sasicodes** [comment]
closes #383

File: src/components/Common/Players/VideoPlayer.tsx [hunk 1]
```diff
@@ -81,6 +81,11 @@ const PlayerInstance = ({ time, source, ratio, hls, poster }: PlayerProps) => {
     })
   }
 
+  useEffect(() => {
+    if (!playerRef.current) return
+    playerRef.current.currentTime = Number(time || 0)
+  }, [playerRef, time])
+
```
Comment (sasicodes) [inline src/components/Common/Players/VideoPlayer.tsx:88]:
```currentVideo.onloadedmetadata = () => {
      currentVideo.currentTime = Number(time || 0)
    }```

If this not required, can remove this.

Comment (RamK777-stack) [reply src/components/Common/Players/VideoPlayer.tsx:88]: This needed for initial render, without this copy at current time share URL not working.!

**Review by sasicodes — APPROVED** [review]

**Comment by sasicodes** [comment]
LGTM 🚀

**Pull request merged**